generators/heroku/index.js

```diff
@@ -756,6 +737,12 @@ module.exports = class extends BaseBlueprintGenerator {
               }
               if (curlAvailable && jqAvailable) {
                 this.log(chalk.green('Running ./provision-okta-addon.sh to create all required roles and users to use with jhipster.'));
+                this.log(
+                  chalk.green.bold(
+                    `Going to provision okta. Use ${this.oktaAdminLogin}/${this.oktaAdminPassword} to login. You will need to change your password afterwards.`
```

Comment:
Is "Going to provision Okta." necessary here? I'd change it to something like:

```
Okta configured successfully! Use ${this.oktaAdminLogin}/${this.oktaAdminPassword} to sign in.
```

Reply:
In fact it is not yet configured successfully, but I agree, it sounds better,

Reply:
I wasn't aware of that. It's best not to give a false positive. Is it possible to emit the user/pass at the end of the process? 

One issue we noticed with the Okta CLI, is that we need to pause (or something) to make sure the user didn't see the password scroll by (and clear the console).

Reply:
Advice from a co-worker:

> I think it's fine, but you might want to prompt a "make note of this password and continue" if possible
> You could also pop a browser with a password reset link

Reply:
How do you generate the password reset link? 

You call our https://developer.okta.com/docs/reference/api/users/#reset-password API.

This returns an object like this:

```json
{
  "resetPasswordUrl": "https://${yourOktaDomain}/reset_password/XE6wE17zmphl3KqAPFxO"
}
```

Then you can replace `reset_password` with `welcome` (so you get our new password reset flow). 

Not sure if this is worth it, but figured I'd mention it.

Reply:
I think the current flow is fine. You login from jhipster and need to change your password directly. No need to do a dedicated reset step imho.